app/tests/unit/utils/localStorage.test.ts

```diff
@@ -89,10 +90,6 @@ describe('localStorage tests', () => {
     vi.resetAllMocks()
   })
 
-  afterEach(() => {
-    localStorage.clear()
-  })
-
   test('localStorage keys snapshot', () => {
```

Comment:
On line 81, I think it would be great to add as a comment what date time does that unix time refer to.